dozer-cli/src/live/state.rs

```diff
@@ -49,6 +50,7 @@ pub struct LiveState {
     run_thread: RwLock<Option<ShutdownSender>>,
     error_message: RwLock<Option<String>>,
     sender: RwLock<Option<tokio::sync::broadcast::Sender<ConnectResponse>>>,
+    temp_dir: RwLock<Option<TempDir>>,
```

Comment:
Why we have to create another `RwLock`? Can we change `run_thread` to `RwLock<Option<ShutdownAndTempdir>>`?

```rust
struct ShutdownAndTempDir {
  shutdown: ShutdownSender,
  temp_dir: TempDir
}
```